compose-development.yml

```diff
@@ -1,4 +1,4 @@
-shareddatadev:
+shareddata:
```

Comment:
because we don't need that duplication of names, since the compose-files are only applicatble for development.